### DISCUSSION TITLE
Drifty is now featured on GitHub Education's Community Exchange

### DISCUSSION THREAD

**Discussion by ericdrosado** [SaptarshiSarkar12/Drifty#343] [Announcements]
👋 Hi @SaptarshiSarkar12 ! I just wanted to let you know that **Drifty** is now featured on GitHub Education's [Community Exchange](https://education.github.com/globalcampus/exchange). 🎉 

Congratulations! Thank you for being an open-source Maintainer!

**Comment by SaptarshiSarkar12** [comment]
Thank you @ericdrosado for the information. I am glad to have my project featured in Community Exchange 🥹.
Thank you, Open-Source, and everyone who supported this project!

**Reply by SaptarshiSarkar12** [reply]
A screenshot of the feature :point_down:
![Screenshot from 2023-11-21 00-28-08](https://github.com/SaptarshiSarkar12/Drifty/assets/105960032/e6fc7671-918b-49e2-89fa-22e55b88fc34)

**Reply by VKS1572** [reply]
Congratulations @ericdrosado and @SaptarshiSarkar12! 🎉
Seeing Drifty featured on GitHub Education’s Community Exchange is truly inspiring.
Huge respect to the maintainers and contributors who helped shape the project. 🚀

Open-source wins like this motivate developers like me to keep contributing and learning.
Looking forward to seeing Drifty grow even more! 🙌

**Reply by SaptarshiSarkar12** [reply]
Thanks so much for the incredibly kind words, @VKS1572! 🙏
It truly means a lot to hear that it is inspiring developers like you. The project's growth is a direct result of the effort from every single contributor. It's the community's support that drives Drifty forward! 💚
